### DISCUSSION TITLE
Request: Make volume slider larger and easier to use on a phone

### DISCUSSION THREAD

**Discussion by fernplant** [music-assistant/support#716] [Feature requests and ideas]
Hi all

I struggle with adjusting speaker volume when on my Android phone. When I try to tap and slide the small "slide knob", it thinks Im trying to slide away the whole speaker view instead. Ive found that its most effective to use a downward and to the side slide to adjust the volume but not make it think you want to change the view. Its still difficult though and not smooth. This is in portrait mode by the way. I hope that this makes sense; if not, I will try to elaborate further

Thanks

**Comment by marcelveldt** [comment]
I can't reproduce this on iOS so maybe specific to android ?

Would it help if I add a plus and minus button here?

**Comment by fernplant** [comment]
Sorry for the delay in getting back on this. I didnt want to mess with getting a screen recorder so took a video of what Im describing. Hopefully it makes more sense with seeing it. View pane sliding action supersedes the volume slide action  and very difficult to slide volume in one direction

Video: https://photos.google.com/share/AF1QipMSEAl5pxkj1H6CE-uPERJzpPUBff8CpYBIvPpM5gOh2x52jFRvuhd4AbJnpZiUcQ?key=YXNnTXdhN1IxV1B4Q3ozLUFPVW81VWF2aHhEQ0JR

**Comment by svenvdmeer** [comment]
I'm really surprised this issue/request doesn't have move votes. This issue makes mass unusable for me. It is very important to easily and reliably control the volume.
Does nobody else have this issue or is there a workaround or something?

**Comment by marcelveldt** [comment]
This is fixed in Music Assistant 2.0 beta 10

**Reply by svenvdmeer** [reply]
Great it works a lot better now. There is still some unwanted movement in the background if you have the left menu open. But this does not interfere with controlling the volume. I've made a screen recording. 

https://photos.app.goo.gl/GUkEeekdN6JEcZDA9

**Reply by marcelveldt** [reply]
fancy :-) 
didn't notice that on iPhone. I'll try to workaround it. At least now the volume slider is controllable.

**Reply by fernplant** [reply]
Awesome, glad to see a fix implemented. Thanks much
